src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -192,6 +192,12 @@
   <data name="Globals_CopyOnSelect.Header" xml:space="preserve">
     <value>Automatically copy selection to clipboard</value>
   </data>
+  <data name="Globals_KeybindingsDisclaimer.Text" xml:space="preserve">
+    <value>This is a list of the currently bound keys. Currently, these can only be edited in the settings file directly.</value>

```

Comment:
"directly" at the end of a sentence feels weird. @cinnamon-msft thoughts on this disclaimer?

Reply:
Maybe `Below are the currently bound keys, which can be modified by editing the JSON settings file.`?